stimpacks/deploy/deploy.go

```diff
@@ -48,6 +49,19 @@ func (d *Deploy) Run() {
 
 	// Read in the config file and set up defaults
 	d.parseConfig()
+	if d.config.Global.RequiredVersion != "" {
+		d.log.Info("Deploy has set RequiredVersion to:{}, currently:{}", d.config.Global.RequiredVersion, d.stim.GetVersion())
```

Comment:
Maybe debug on this one?

Reply:
I feel like it should be logged, esp to start with since people need to see this is needed w/o being in debug.  I get that we will bail out if it does not match, and log then, but adding this line will help people when they are adding it to the stim.deploy.yaml for the first time.  We can move it to debug later if needed but I would like to keep it for now.